.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    ["@babel/env", {
```

Comment:
why you gotta do me like that?

Reply:
your browser support is too restrictive :p